This fixes bug in TZ build that fails with perms

ericvh opened the pull request:
Sets USER to root in the dockerfile for portion of docker build
for dpkg and apt command section

Signed-off-by: Eric Van Hensbergen <eric.vanhensbergen@arm.com>

gbryant-arm commented on Makefile:
```diff
@@ -37,8 +36,20 @@ else # otherwise linux
 .PHONY:
 build: Dockerfile
+ifeq ($(TEE),nitro)
+	git clone https://github.com/aws/aws-nitro-enclaves-cli.git
```
We might want to fail silently in case the repo already exists.
`git clone https://github.com/aws/aws-nitro-enclaves-cli.git || echo "failure"` for instance?

dreemkiller replied:
I've been thinking about what to do with this. Currently if it already exists, it errors out (not silently).
Ideally, it would not error out if it exists, but instead pull updates, and only rebuild if there are changes. 

That's a bit more than I want to bite off right now, but yes, something should be done.

dreemkiller commented:
This is ready to review. It's a partner to the https://github.com/veracruz-project/veracruz/pull/90 in the veracruz repo. They should be merged together.

ericvh committed:
This fixes bug in TZ build that fails with perms

Sets USER to root in the dockerfile for portion of docker build
for dpkg and apt command section

Signed-off-by: Eric Van Hensbergen <eric.vanhensbergen@arm.com>

dominic-mulligan-arm commented:
Is this still meant to be a draft or is it complete?

ericvh commented:
I was waiting for the test to complete.  There are some other fixes but
they are unrelated to the original issue so I'll mark this as complete for
now and submit the others elsewhere.


On Fri, May 14, 2021 at 7:50 AM Dominic Mulligan ***@***.***>
wrote:

> Is this still meant to be a draft or is it complete?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/veracruz-project/veracruz-docker-image/pull/10#issuecomment-841223303>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAAQ7WGGPKYMWVT2PVJDVILTNUMCXANCNFSM44226SDQ>
> .
>

dominic-mulligan-arm commented:
+1 reached, merging.
